### ISSUE TITLE
Add Python3.x support

### ISSUE THREAD

**Issue by jsh2134** [solvebio/veppy#11]
a full python3.x audit was never done on this. likely there are a lot of simple fixes needed in order to be compatible, and a few big ones :)

See failed builds https://travis-ci.org/solvebio/veppy/builds/165497165

**Comment by gsrajadh** [comment]
Can I work on this?

**Comment by jsh2134** [comment]
yes! that would be great. please create a new branch called py3-support. you can get started by running flake8 on veppy to see what doesn't match. travis is uncovering some of those now https://travis-ci.org/solvebio/veppy/jobs/165520902#L167

also see https://docs.python.org/3/howto/pyporting.html

**Comment by gsrajadh** [comment]
@jsh2134 I will start working on it today. Thanks!
